library/neutron_network.py

```diff
@@ -18,6 +18,8 @@
 
 try:
     from neutronclient.neutron import client
+    from keystoneauth1.identity import v2
```

Comment:
You mean the keystoneauth1 as v2? It's a weird naming thing. The keystoneauth1 library provides the API v2 stuff.